mantidimaging/gui/widgets/spectrum_widgets/fitting_display_widget.py

```diff
@@ -57,6 +58,10 @@ def update_plot(self,
     def update_image(self, image: np.ndarray | None) -> None:
         if image is not None:
             self.image_item.setImage(image, autoLevels=True)
+            height, width = image.shape
+            scale_x = 150 / width
+            scale_y = 150 / height
+            self.image_item.setTransform(QTransform().scale(scale_x, scale_y))
```

Comment:
Can this be
```python
            scale = 150 / max(width, height)
            self.image_item.setTransform(QTransform().scale(scale, scale))
```

So that it preserves aspect ratio